stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/extensions/ChatClient.kt

```diff
@@ -74,7 +74,7 @@ public val ChatClient.state: StateRegistry
  * [GlobalState] instance that contains information about the current user, unreads, etc.
  */
 public val ChatClient.globalState: GlobalState
-    get() = GlobalMutableState.getOrCreate()
+    get() = GlobalMutableState.get()
```

Comment:
🤔 why are we changing this? Can we be sure `GlobalState` is already initialized at this point? If yes, maybe we don't need `GlobalState::getOrCreate` at all?